app_legacy/Helpers/render/game.php

```diff
@@ -56,25 +56,93 @@ function gameAvatar(
 /**
  * Render game title, wrapping categories for styling
  */
-function renderGameTitle(?string $title): string
+function renderGameTitle(?string $title, bool $tags = true): string
 {
+    // Update $html by appending text
+    $updateHtml = function (&$html, $text, $append) {
+        $html = trim(str_replace($text, '', $html) . $append);
+    };
+
     $html = (string) $title;
     $matches = [];
     preg_match_all('/~([^~]+)~/', $title, $matches);
     foreach ($matches[0] as $i => $match) {
         $category = $matches[1][$i];
         $span = "<span class='tag'><span>$category</span></span>";
-        $html = trim(str_replace($match, '', $html) . ' ' . $span);
+        $updateHtml($html, $match, $tags ? " $span" : '');
     }
-    preg_match_all('/\[(Subset - (.+))\]/', $title, $matches);
-    foreach ($matches[0] as $i => $match) {
-        $subset = $matches[2][$i];
-        $span = "<span class='tag'>";
-        $span .= "<span class='tag-label'>Subset</span>";
-        $span .= "<span class='tag-arrow'></span>";
-        $span .= "<span>$subset</span>";
-        $span .= "</span>";
-        $html = trim(str_replace($match, '', $html) . ' ' . $span);
+    $matches = [];
+    if (preg_match('/\[Subset - (.+)\]/', $title, $matches)) {
+        $subset = $matches[1];
+        $span = "<span class='tag'>"
+            . "<span class='tag-label'>Subset</span>"
+            . "<span class='tag-arrow'></span>"
+            . "<span>$subset</span>"
+            . "</span>";
+        $updateHtml($html, $matches[0], $tags ? " $span" : '');
+    }
+
+    return $html;
+}
+
+/**
+ * Render game breadcrumb prefix, with optional link on last crumb
+ *
+ * Format: `All Games » (console) » (game title)`.
+ * If given data is for a subset, then `» Subset - (name)` is also added.
+ */
+function renderGameBreadcrumb(array $data, bool $addLinkToLastCrumb = true)
+{
+    // Return next crumb (i.e `» text`), either as a link or not
+    $nextCrumb = function ($text, $href = ''): string {
+        return " &raquo; " . ($href ? "<a href='$href'>$text</a>" : "<b>$text</b>");
+    };
+
+    // Retrieve separete IDs and titles for main game and subset (if any)
+    $getSplitData = function ($gameID, $gameTitle): array {
+        $matches = [];
+        if (preg_match('/(.+)(\[Subset - .+\])/', $gameTitle, $matches)) {
+            [$gameTitle, $subset] = [trim($matches[1]), $matches[2]];
+            $subsetID = $gameID;
+            $renderedSubset = renderGameTitle($subset);
+
+            // Retrieve main game ID
+            sanitize_sql_inputs($gameID);
+            settype($gameID, 'integer');
+            $query = "SELECT ga.gameIDAlt FROM GameAlternatives ga
+                WHERE ga.gameID = $gameID";
+            $result = s_mysql_query($query);
+            $gameID = $result->fetch_array()[0];
```

Comment:
This query probably needs some adjustments. Several subsets are linked to more than one game alternative - especially if you include the hubs. It's just happenstance that the first linked game is usually the parent. I recommend stripping the [Subset] moniker and searching for the game title. This could be combined with the logic below for determining if a base title is shared.

Reply:
> It's just happenstance that the first linked game is usually the parent

Yep, I (wrongly) assumed that was the rule and not merely usual.